registrations/models.py

```diff
@@ -525,6 +554,11 @@ class PresenceStatus:
         default=False,
     )
 
+    anonymization_time = models.DateTimeField(
+        verbose_name=_("Anonymization time"),
+        null=True,
+    )
+
```

Comment:
```suggestion
    anonymization_time = models.DateTimeField(
        verbose_name=_("Anonymization time"),
        null=True,
        editable=False,
    )

```

Reply:
Added `editable=False` to `anonymization_time` field